pipeline-model-definition/src/test/resources/envVarContainsTEST.groovy

```diff
@@ -37,6 +37,8 @@ pipeline {
             steps {
                 echo "TEST_VAR is ${TEST_VAR}"
                 echo "VAR_TEST is ${VAR_TEST}"
+                sh 'echo "TEST_VAR from shell is ${TEST_VAR}"'
+                sh 'echo "VAR_TEST from shell is ${VAR_TEST}"'
```

Comment:
Well, the `${}` are valid in bash but in this case unnecessary.
It might cause confusion that groovy is expected to expand it, at least I had to read it three times to get that :)